assets/src/edit-story/components/colorPicker/colorPicker.js

```diff
@@ -187,9 +180,7 @@ function ColorPicker({
               onRotate={rotateClockwise}
               onMove={moveCurrentStopBy}
             />
-          </Body>
-        )}
-        <Body>
+          )}
           <CurrentColorPicker
             color={currentColor}
```

Comment:
I don't think the relevant piece of code is in this PR, as it probably also exists on main, but this line is vaguely relevant.

The `currentColor` is not correctly reset when switching from one gradient to another using the saved colors. Look at this example where I select a saved gradient with only a purple and blue stop, but the current color display is still showing a green color (leftover from the old gradient), even though the currently selected stop in the gradient line is clearly purple:

![gradient-display-off](https://user-images.githubusercontent.com/637548/108577263-66f44700-72ee-11eb-8317-b810c3275439.gif)

Reply:
Added an issue for it: https://github.com/google/web-stories-wp/issues/6478

Indeed, the current color doesn't seem to change specifically in case of gradients.